source/Armament.cpp

```diff
@@ -164,7 +164,8 @@ void Armament::Swap(int first, int second)
 
 	// Swap the weapons in the two hardpoints.
 	const Outfit *outfit = hardpoints[first].GetOutfit();
-	hardpoints[first].Install(hardpoints[second].GetOutfit());
+	const Outfit *secondOutfit = hardpoints[second].GetOutfit();
+	hardpoints[first].Install(secondOutfit);
```

Comment:
This doesn't really change anything.

Reply:
This was a mistake.  You're right it was a failed experiment that I aborted.  I'll revert this.